README.md

```diff
@@ -19,6 +20,34 @@
 * [Replacing DataPusher with XLoader](#Replacing-DataPusher-with-XLoader)
 
 
+## 0. Getting Started Quickly
+
+These instructions are for a quick CKAN Development environment running on docker:
+
+Clone the Repository and go to the root directory.
+
+	git clone https://github.com/CivityNL/ckan-docker.git
+    cd ckan-docker
+
+Checkout `civity` Branch
+
+    git checkout civity
+
+Create a copy of `.env.civity` file next to it and rename it to `.env`
+
+	cp .env.civity .env
```

Comment:
If we really want to have it quick, shouldn't we just rename this `.env.civity` to `.env`? In that case this step is obsolete

Reply:
keep in line with community .gitignore